config.yml

```diff
@@ -1,4 +1,5 @@
-name: "Suicid Sqad"
+name: "Lowlighter"
+signature: "Made with <span style='color:pink;'><3</span><br>©2021. All Rights Reserved"
```

Comment:
Maybe use 💕, ❤️ or 💖 to make it 21<sup>st</sup> century rather than 20<sup>th</sup> 🙂 ? 

Not sure yet, but maybe we should support pre-formatted strings out of the box using `white-space: pre` so users could directly use:
```yaml
signature: |
    Made with 💖
    ©2021. All Rights Reserved
```
Seems nice to allow formatting without any html, it makes it more readable.

Reply:
I don't like emojis. I prefer old school ascii art instead. but nvm, gonna change it asap.

Reply:
commit a8fd6bfef4aa511f9eb5dc2ff337ed777f79348c